### PR TITLE
Correct value setter tests

### PR DESCRIPTION
**Pull request by TortoiseWrath**

### CODE REVIEW

**Comment by micah-white** [comment]
I'm not sure why you did this. I wanted to test both fields

**Comment by TortoiseWrath** [comment]
@micah-white I assume you mean both fields in the twitter item (username and userId). They are still both tested by the "updating twitter using field should update json" - the point of the second test "updating twitter using value setter should update json," which had been identical to the first test, is to test the setter `value` rather than direct field access. (The `value` setter takes a user-entered value which should not be passed directly into the field.) In the case of Twitter, that setter only directly affects the username field in a predictable way; the ID will then be detected from the supplied username.

**Comment by TortoiseWrath** [comment]
oh you were confused because I fucked it up in my own tests

File: test/items/email_test.dart [hunk 1]
```diff
@@ -23,7 +23,7 @@ void main() {
       expect(item.toJson()['email'], 'abc@ijk.xyz');
     });
     test('updating email using value setter should update json', () {
-      item.email = 'def@ijk.xyz';
+      item.value = 'def@ijk.xyz';
```
Comment (TortoiseWrath) [inline test/items/email_test.dart:26]: changed here as well (my test)

**Pull request merged**